lean-toolchain

```diff
@@ -0,0 +1 @@
+leanprover/lean4:nightly-2022-06-08
```

Comment:
```suggestion
leanprover/lean4:nightly-2022-06-08

```

Reply:
Hm, I think Lake should be happy without the newline but I'll add just in case.